examples/ipv6/rpl-udp/udp-client.c

```diff
@@ -40,9 +40,7 @@
 #include <stdio.h>
 #include <string.h>
 
-/* Only for TMOTE Sky? */
 #include "dev/serial-line.h"
-#include "dev/uart1.h"
```

Comment:
Yes, this example should be platform-independent, thus I don't like this approach... normally the `serial-line-init` is done at platform's initialisation:

````
$ grep -lr "serial_line_init(" .
./srf06-cc26xx/contiki-main.c
./openmote-cc2538/contiki-main.c
./nrf52dk/contiki-main.c
./zoul/contiki-main.c~
./zoul/dev/mp3-grove.c~
./zoul/contiki-main.c
./eval-adf7xxxmb4z/contiki-main.c
./cc2538dk/contiki-main.c
./cooja/contiki-cooja-main.c
./jn516x/contiki-jn516x-main.c
./cooja-ip64/contiki-cooja-ip64-main.c
./ev-aducrf101mkxz/contiki-main.c
./mbxxx/contiki-main.c
./wismote/contiki-wismote-main.c
./native/contiki-main.c
./avr-rss2/contiki-main.c
./z1/contiki-z1-main.c~
./z1/contiki-z1-main.c
./sky/contiki-sky-main.c
./exp5438/contiki-exp5438-main.c
./cc2530dk/contiki-main.c
````